provider/resources.go

```diff
@@ -2292,6 +2297,42 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_codeartifact_repository_permissions_policy": {
 				Tok: awsResource(codeartifactMod, "RepositoryPermissionsPolicy"),
 			},
+
+			// Imagebuilder
+			"aws_imagebuilder_component": {Tok: awsResource(imageBuilderMod, "Component")},
+			"aws_imagebuilder_distribution_configuration": {
+				Tok: awsResource(imageBuilderMod, "DistributionConfiguration"),
+			},
+			"aws_imagebuilder_infrastructure_configuration": {
+				Tok: awsResource(imageBuilderMod, "InfrastructureConfiguration"),
+			},
+
+			// Network firewall
+			"aws_networkfirewall_firewall": {Tok: awsResource(networkFirewallMod, "Firewall")},
+			"aws_networkfirewall_firewall_policy": {
+				Tok: awsResource(networkFirewallMod, "FirewallPolicy"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"firewall_policy": {
+						CSharpName: "FirewallPolicyConfiguration",
+					},
+				},
+			},
+			"aws_networkfirewall_logging_configuration": {
+				Tok: awsResource(networkFirewallMod, "LoggingConfiguration"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"logging_configuration": {
+						CSharpName: "LoggingConfig",
+					},
+				},
+			},
+			"aws_networkfirewall_rule_group": {
+				Tok: awsResource(networkFirewallMod, "RuleGroup"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"rule_group": {
+						CSharpName: "RuleGroupConfiguration",
```

Comment:
I don't think we have a guideline about the naming here. I would stay less creative and appended `Value` to every conflicting name in C#. Not sure how strong you are about `Configuration`.

Reply:
So I added Configuration as it's a config block... usually I am much less aggressive with these names